community/teach_me_qiskit_2018/elementary_arithmetic_operations/elementary_arithmetic_operations.ipynb

```diff
@@ -41,7 +48,7 @@
     "With this figure we seek a generalization of the algorithm, but the first carry qubit $c_0$ is not really needed, because the value will always be $0$.\n",
     "\n",
     "### Implementation in ibmqx2. Example 1.\n",
-    "We are going to start with the easiest example, the implementation of the plain addition of two one-qubit numbers $a$ and $b$. This program can be executed in ibmqx2 (5 qubits)."
+    "We are going to start with the easiest example, the implementation of the plain addition of two one-qubit numbers $a$ and $b$. This program can be executed a 5 qubits device."
```

Comment:
`a 5-qubit device` to be grammatically correct?

Reply:
Yes, I will change that.